### PR TITLE
add fetch

### PR DESCRIPTION
**Pull request by dawidzon**
Testowy pull request.

Dodałem dane poprzez fetch.

https://user-images.githubusercontent.com/53623545/202899423-93a4d8f5-3a07-44e9-b211-e12384e957d0.mov

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -35,6 +35,26 @@ function render() {
...
+          articles.push({
+            id: articles.length + 1,
+            title: el.title,
+            body: el.body,
+            likesCount: 0,
+          });
+          render();
+        })
+      );
+  };
```
Comment (wozny1989) [inline main.js]: It's completely fine! 🚀 But for challange, try move this function to `async`/`await` approach
```suggestion
  addPostsFromOutside.onclick = async () => {
    const result = await fetch("https://jsonplaceholder.typicode.com/posts?userId=1");
    // ToDo...
  };
```

**Review by wozny1989 — CHANGES_REQUESTED** [review]
@dawidzon One thing 💪

File: main.js [hunk 2]
```diff
@@ -35,6 +35,28 @@ function render() {
...
+        result.forEach((el) => {
+          articles.push({
+            id: articles.length + 1,
+            title: el.title,
+            body: el.body,
+            likesCount: 0,
+          });
+          render();
+        })
+      );
```
Comment (wozny1989) [inline main.js]: Nope, use [google search ](https://www.google.com/search?q=fetch+async+await&rlz=1C5CHFA_enPL1028PL1028&oq=fetch+async+await&aqs=chrome.0.0i512l8j0i390l2.2726j0j1&sourceid=chrome&ie=UTF-8) if don't know what's I mean :)

**Review by wozny1989 — CHANGES_REQUESTED** [review]

**Comment by wozny1989** [comment]
@dawidzon By the way, use english description as well. You wouldn't want your future client to see this in this form 😄

**Comment by dawidzon** [comment]
@wozny1989  - thank you for the hint.

when it comes to async / await, if it's still not OK, then I don't know how to use it xD

File: main.js [hunk 3]
```diff
@@ -35,6 +35,27 @@ function render() {
...
+
+    data.forEach((el) => {
+      articles.push({
+        id: articles.length + 1,
+        title: el.title,
+        body: el.body,
+        likesCount: 0,
+      });
+      render();
+    });
```
Comment (wozny1989) [inline main.js]:
```suggestion
    data.forEach((article) => {
      articles.push({
        id: articles.length + 1,
        title: article.title,
        body: article.body,
        likesCount: 0,
      });
      render();
    });
```

or more fancy

```suggestion
    data.forEach(({ title, body }) => {
      articles.push({
        id: articles.length + 1,
        title,
        body,
        likesCount: 0,
      });
      render();
    });
```

**Review by wozny1989 — CHANGES_REQUESTED** [review]

**Comment by dawidzon** [comment]
I worked on another branch with login page and forgot to pull version when I change back to fetch branch, becouse of that in this version I add some changes connected with login page.

**Review by wozny1989 — APPROVED** [review]
Perfect 👌

**Comment by wozny1989** [comment]
@dawidzon Merged, please switch locally to the `main` branch and pull the latest changes

**Pull request merged**